packages/support/src/settings/_variables.shadow.scss

```diff
@@ -8,3 +8,8 @@ $shadow-base-blur-radius: 4px;
 $shadow-base-color: rgba(0, 0, 0, 0.25);
 $shadow-base: $shadow-base-offset-x $shadow-base-offset-y $shadow-base-blur-radius
   $shadow-base-color;
+
+// Inherited from USWDS 
+$box-shadow: 0 0 2px $shadow-base-color !default;
```

Comment:
We should standardize these in a future PR

Reply:
Is there a work item for this future work?

Reply:
not yet! ideally it could be sorted out after the focus PR

Reply:
Actually, once this PR is merged in to master we can handle it in https://github.com/CMSgov/design-system/pull/510